app/src/main/java/com/example/lhi06/residue/AlbumAdapter.java

```diff
@@ -77,9 +77,9 @@ public int getItemCount() {
     }
 
     public void removeAt(@NonNull final int position) {
-        albumList.remove(position);
+        albums.remove(position);
         notifyItemRemoved(position);
-        notifyItemRangeChanged(0, albumList.size());
+        notifyItemRangeChanged(0, albums.size());
```

Comment:
And you shouldn't be copying stuff from tutorials that you don't really understand :)